genestack_client/file_filters.py

```diff
@@ -185,3 +185,21 @@ class OrFileFilter(FileFilter):
     def __init__(self, first, second):
         super(OrFileFilter, self).__init__()
         self._dict.update({'or': [first.get_dict(), second.get_dict()]})
+
+
+class MetainfoRelatedValueFilter(FileFilter):
+    """
+    Filter to select files with metainfo values related to given terms by a dictionary relationship.
```

Comment:
This API is actually 'more' public than the Java API, so it its docsting must be at least as big as the javadoc. I'd also add some usage examples.